pages/sign-up.js

```diff
@@ -54,46 +77,38 @@ const SignUp = () => {
 
                         <Input label="Confirm Password" name="confirmPassword" type="password" />
 
-                        <div className="flex items-center mt-8 mb-3">
-                          <input
-                            id="remember-me"
-                            name="remember-me"
-                            type="checkbox"
-                            placeholder="Your password"
-                            className="w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 "
+                        <div className=" mt-8 mb-3">
+                          <CheckBoxGroup name="acceptMulti" options={output} />
+                          <Checkbox
+                            name="acceptTerms"
+                            label={
+                              <label>
+                                I have read, and I agree to the
+                                <a href="#" className=" text-amcovad-primary-400 hover:text-amcovad-primary-500">
+                                  {' '}
+                                  Terms of Service{' '}
+                                </a>
+                                and{' '}
+                                <a href="#" className=" text-amcovad-primary-400 hover:text-amcovad-primary-500">
+                                  Privacy Policy
+                                </a>
+                              </label>
+                            }
                           />
-                          <label
-                            htmlFor="remember-me"
-                            className="block ml-2 text-[0.75rem] text-[#344055] font-normal font-Inter "
-                          >
-                            I have read, and I agree to the
-                            <a href="#" className=" text-amcovad-primary-400 hover:text-amcovad-primary-500">
-                              {' '}
-                              Terms of Service{' '}
-                            </a>
-                            and{' '}
-                            <a href="#" className=" text-amcovad-primary-400 hover:text-amcovad-primary-500">
-                              Privacy Policy
-                            </a>
-                          </label>
                         </div>
                         <div className="flex items-center mt-3 mb-6">
-                          <input
-                            id="remember-me"
-                            name="remember-me"
-                            type="checkbox"
-                            className="w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 "
+                          <Checkbox
+                            name="agreeContact"
+                            label={
+                              <label>
```

Comment:
Also, you don't need the label tag here since it is defined in the Checkbox component.

```suggestion
                              <>
```